leftwm/src/config.rs

```diff
@@ -174,11 +174,29 @@ impl WindowHook {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub enum Backend {
+    #[cfg(feature = "xlib")]
+    XLib,
+    #[cfg(feature = "x11rb")]
+    X11rb,
+}
+
+impl Default for Backend {
+    fn default() -> Self {
+        #[cfg(feature = "xlib")]
+        return Backend::XLib;
+        #[cfg(not(feature = "xlib"))]
```

Comment:
this can probably be just `#[cfg(feature = "x11rb")]`

Reply:
ok no just forget this i'm tired i didn't look at the context xD

Reply:
Yeah it's also the reason the Default implementation isn't just the derive macro.